gapir/cc/interpreter.cpp

```diff
@@ -76,8 +76,15 @@ Interpreter::Interpreter(core::CrashHandler& crash_handler,
                     stack->printStack();
                     return true;
                   });
+
+  mRendererFunctions = new FunctionTable*[16];
```

Comment:
Why allocate this on the heap if its always going to be the same size?
FunctionTable* mRendererFunctions[16] = {};

Reply:
Done.